src/backend/InvenTree/report/models.py

```diff
@@ -42,6 +44,31 @@
 
 logger = structlog.getLogger('inventree')
 
+WE_BASE_URL = 'http://localhost'
```

Comment:
An issue here is that the request URL may or may not exactly match the server's internal URL, depending on proxying settings. Adding a hard-coded URL here is not a good idea.